syft/execution/role.py

```diff
@@ -171,14 +171,39 @@ def execute(self):
 
         return tuple(p.child for p in output_placeholders)
 
+    def store(self, tensor):
+        """ store tensors used in a protocol to worker's local store
+        """
+        if "<class 'torch.Tensor'>" == str(type(tensor)):
```

Comment:
I think we'll also want to store other things than tensors. Parameters or stuff like that for instance.